### ISSUE TITLE
Set up skeleton project

### ISSUE THREAD

**Issue by sdruskat** [hermes-hmc/workflow#3]
Before we start the implementation proper, we should set up a skeleton project.

- [x] pyproject.toml
- [x] folder structure
- [x] README
- [x] ~~LICENSE~~ deferred to #4
- [x] ~~CONTRIBUTING.md?~~ deferred to #5
- [x] ~~CoC?~~ deferred to #6
- [x] test env (pytest)
- [x] ~~docs~~ deferred to #7
